src/Raven.Studio/typescript/viewmodels/database/documents/editDocument.ts

```diff
@@ -1438,8 +1438,11 @@ class editDocument extends shardViewModelBase {
         this.confirmationMessage("Are you sure?", "Do you want to delete current revision?", {
             buttons: ["Cancel", "Yes, delete"],
         }).done((result) => {
+            const docsIds = parameters.DocumentIds.join(", ")
+
             if (result.can) {
                 new deleteRevisionsForDocumentsCommand(this.db.name, parameters).execute().done(() => {
+                    messagePublisher.reportSuccess(`Deleted ${docsIds}`)
```

Comment:
You have many revisions for one doc id. Maybe include change vector in message, e.g. `Deleted revision ${doc.__metadata.changeVector()}`